zwave_js_server/model/node/__init__.py

```diff
@@ -901,24 +909,14 @@ def handle_value_updated(self, event: Event) -> None:
         if value is None:
             value = _init_value(self, evt_val_data)
             self.values[value.value_id] = event.data["value"] = value
-            self.data["values"].append(evt_val_data)
         else:
             value.receive_event(event)
             event.data["value"] = value
-            self.data["values"][self.value_data_idx(value_id)].update(evt_val_data)
-
-        node_val_data = self.data["values"][self.value_data_idx(value_id)]
-        if "newValue" in evt_val_data:
```

Comment:
Is this `newValue` item handled in the property `data` that we now expose?

Reply:
no longer relevant since we removed the new property